File: xwallpaper/src/main/java/vn/loitp/app/activity/api/truyentranhtuan/model/chap/Chap.java

package vn.loitp.app.activity.api.truyentranhtuan.model.chap;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Chap {

    @SerializedName("tit")
    @Expose
    private String tit;

    @SerializedName("url")
    @Expose

    private String url;

    /**
     * @return The tit
     */
    public String getTit() {
        return tit;
    }

    /**
     * @param tit The tit
     */
    public void setTit(String tit) {
        this.tit = tit;
    }

    /**
     * @return The url
     */
    public String getUrl() {
        return url;
    }

    /**
     * @param url The url
     */
    public void setUrl(String url) {
        this.url = url;
    }

}

File: core/src/main/java/com/core/helper/ttt/model/chap/Chap.kt
package com.core.helper.ttt.model.chap

import androidx.annotation.Keep
import com.core.base.BaseModel
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Keep
class Chap : BaseModel() {
    @SerializedName("tit")
    @Expose
    var tit: String = ""

    @SerializedName("url")
    @Expose
    var url: String = ""

}
